FTP: Replace menu (25.2) [STUD-73198]

viogroza opened the pull request:
Make the toggle between string and secure string
At runtime use the actual mode instead of using password and if not set fallback to secure password

https://uipath.atlassian.net/browse/STUD-73198

alexandru-petre commented on Activities/FTP/UiPath.FTP.Activities/NetCore/ViewModels/WithFtpSessionViewModel.cs:
```diff
@@ -135,12 +135,12 @@ protected override void InitializeModel()
               .AddMenuProperty(ClientCertificatePassword, PasswordInputMode.Password)
               .AddMenuProperty(ClientCertificateSecurePassword, PasswordInputMode.SecurePassword)
-              .BuildAndInsertMenuActions();
+              .BuildAndInsertMenuActions(false);
```
I think that this one, although not mentioned explicitly in the ticket, suffers from the same lack of visibility and should be promoted to a main option as well. We should ask Maciej.

viogroza replied:
I reached to Maciej and he confirmed that the changes should be made also for ClientCertificate
I will update this PR

viogroza committed:
FTP: Replace menu (25.2) [STUD-73198]

Make the toggle between string and secure string
At runtime use the actual mode instead of using password and if not set fallback to secure password
